docs/webrtc/nanostream_webrtc_quality.md

```diff
@@ -0,0 +1,96 @@
+---
+id: nanostream_webrtc_quality
+title: Quality Settings
+sidebar_label: Quality Settings
+---
+
+# Configuring stream quality
+
+Webcasting quality can be configured by setting different stream encoding and transcoding properties.
+In the nanostream Webcaster this is basically performed in three steps:
```

Comment:
Shouldn't it be `nanoStream` instead of `nanostream`?

Reply:
updated